PLUB/Sources/Views/Home/RecruitmentFilterViewController.swift

```diff
@@ -25,16 +26,18 @@ enum RecruitmentFilterSection: CaseIterable {
 }
 
 protocol RecruitmentFilterDelegate: AnyObject {
-  func didTappedConfirmButton(request: CategoryMeetingRequest)
+  func didTappedConfirmButton(request: CategoryMeetingRequest) // 모집 필터를 위한 Request데이터를 전달하기위함
 }
 
 final class RecruitmentFilterViewController: BaseViewController {
   
+  // MARK: - Properties
   weak var delegate: RecruitmentFilterDelegate?
```

Comment:
mark와 같은 주석은 한 줄 띄어 작성해주시겠나요? 저희가 컨벤션을 그렇게 정했어서요. ☺️
```suggestion
  // MARK: - Properties
  
  weak var delegate: RecruitmentFilterDelegate?
```